modules/rosa-sdpolicy-platform.adoc

```diff
@@ -68,6 +68,24 @@ ifndef::openshift-rosa-hcp[]
 endif::openshift-rosa-hcp[]
 clusters at this time. However, custom labels are supported when creating new machine pools.
 
+[id="rosa-sdpolicy-node-lifecycle_{context}"]
+== Node lifecycle
+
+Worker node persistence is not guaranteed and may be replaced at any time as part of the normal operation and management of OpenShift.
+
+Several events affect the longevity of a {product-title} worker node:
+
+* Machine health checks are deployed and configured to ensure that a worker node with a `NotReady` status is replaced to ensure smooth operation of the cluster.
+* AWS EC2 instance retirement may occur at any time when AWS detects irreparable failure of the underlying hardware that hosts the instance.
+ifdef::openshift-rosa[]
+* During upgrades, a new node is provisioned to account for any loss of cluster resources during the upgrade process. Once an upgrade is complete, an older node is then removed from the cluster.
```

Comment:
Suggested adjustment: "During upgrades, a new node is first provisioned to account for any loss of cluster resources during the upgrade process. Once this new node has been successfully integrated into the cluster via automated health checks (described above), an older node is then removed from the cluster."